src/components/Contacts/Contacts.module.css

```diff
@@ -0,0 +1,42 @@
+.wrap {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    width: 100%;
+}
+
+.title {
+    font-family: 'Courier New', Courier, monospace;
+    font-weight: 300;
+    font-size: 30px;
+    margin-bottom: 15px;
+}
+
+.content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.contact {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    text-align: left;
+    font-family: 'Courier New', Courier, monospace;
+    font-weight: 300;
+    font-size: 30px;
+}
+
+.icon {
+    display: block;
+}
+
+.link {
+    color: black;
+    font-family: 'Courier New', Courier, monospace;
+    font-weight: 300;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/ReactProject_karolina/pullRequest?prid=6313319)